src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -321,6 +321,16 @@ protected function createFormBuilder($data = null, array $options = array()): Fo
         return $this->container->get('form.factory')->createBuilder(FormType::class, $data, $options);
     }
 
+    /**
+     * Creates and returns a named form builder instance.
+     *
+     * @final since version 4.1
```

Comment:
just `@final` (we should add the version only for existing methods that are *made* final)